frontend/src/config/constants/navigationPath.tsx

```diff
@@ -0,0 +1,10 @@
+const path = {
+  signIn: 'signin',
+  signUp: 'signup',
+  myCourses: 'mycourses',
+  myProfile: 'profile',
+  requests: 'requests',
+};
+
+const { signIn, signUp, myCourses, myProfile, requests } = path;
+export { signIn, signUp, myCourses, myProfile, requests };
```

Comment:
What is it? We already have some routing in v1-003 brunch. Reuse them please

Reply:
I had created it before merge v003 into develop branch

Reply:
Fixed it add - signIn into this file